source/ci_source/providers/_tests/_buddyWorks.test.ts

```diff
@@ -61,7 +61,7 @@ describe(".isPR", () => {
 describe(".pullRequestID", () => {
   it("pulls it out of the env", () => {
     const buddyWorks = new BuddyWorks(correctEnv)
-    expect(buddyWorks.pullRequestID).toEqual("1799")
+    expect(buddyWorks.pullRequestID).toEqual("pull/1799")
```

Comment:
This will need you to return the number "42" - so I'd check maybe for "pull" in the string and extract the number out in the BuddyWorks class 👍

Reply:
I wasn't aware of that, it seemed to be working just fine. I'm almost certain there's another env variable which holds just the number. I'll update the PR in a few hours @orta
Thanks for review!

Reply:
Updated now @orta

Reply:
BTW I was wondering if it made sense to change the return type of `get pullRequestID` to `number`? :) This way it would be clear just by looking at the definition that a number is required.

Reply:
Maybe it was being set from somewhere else. The GitHub API takes strings for the pull ID - likely legacy on their side, but I'm open to making them numbest so long as it's parsed back into a string in the API layers